robottelo/ui/settings.py

```diff
@@ -13,6 +13,19 @@ def navigate_to_entity(self):
         """Navigate to Settings entity page"""
         Navigator(self.browser).go_to_settings()
 
+    def remove_parameter(self, tab_locator, param_name):
+        """Removes the value of selected parameter under settings"""
+        self.click(tab_locator)
+        loc = locators['settings.edit_param']
+        self.click(loc % param_name)
+        # if parameter is already empty
```

Comment:
why do you need to call method if parameter is already empty?
Probably, it is better to improve test logic rather than have that condition at all?